scheduler/README.asc

```diff
@@ -9,6 +9,10 @@ You can learn more about the algorithms used by Cook in this talk by Li Jin at M
 
 You can build the final jar by running `lein uberjar`.
 
+## Unit Tests
+
+You can run the Cook scheduler unit tests with `lein test`. If you want to see log output on the console, you can use the `test-console` profile, e.g. `lein with-profile +test-console test`.
+
```

Comment:
Can we add a comment that individual namespaces need to require `cook.test.testutil` to view the log in the console.